encoders/firebase-encoders-reflective/src/main/java/com/google/firebase/encoders/reflective/ObjectDecoderProvider.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.encoders.reflective;
+
+import androidx.annotation.NonNull;
+import com.google.firebase.decoders.ObjectDecoder;
+
+interface ObjectDecoderProvider {
```

Comment:
Looks like you're not using it for anything, consider removing altogether

Reply:
I was trying to align with the behaviour of `ReflectiveEncoder`, but it turns out it's redundant. I will remove them.